packages/core/src/components/Checkbox/CheckboxGroup.tsx

```diff
@@ -76,7 +76,7 @@ export const SEED_DATA = {
     fieldName: createFieldNameProp({
       defaultValue: "checkboxGroupValue",
       handlerPropName: "onValueChange",
-      valuePropName: "checkboxGroupValue",
```

Comment:
This was just a bug, the underlying component actually expects a `value` prop already